.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,26 @@
+name: CIFuzz
```

Comment:
please add a comment block to the top of the file explaining what this file is for, and link to relevant websites & docs for it